mandatory/2-function-creation.js

```diff
@@ -20,12 +28,6 @@ function formatPercentage(num) {
   } else {
     return `100%`;
   }
```

Comment:
Would it be a good idea to use logical operator (&& for this exercise) instead of nested if conditionals? In order to make the code easier to understand. Using a code similar to the commented out below?

Reply:
Hi Omer,  I agree with you about the use of logical operator, that surely makes the code more concise and better. I was practicing to follow the instruction step by step. I do struggle to think towards the solutions. So following a mentor's suggestion i tried both and  kept both solutions here, but commented the shorter one to review later. Thank you for your comment. I will  change the code shortly. Thank you again.

Reply:
Ahh I see, good idea both implementing nested if conditions and logical operator to digest them both. Congratulations again, great job 💯